src/database/mod.rs

```diff
@@ -160,35 +160,6 @@ impl ConnectOptions {
         }
     }
 
-    #[cfg(feature = "sqlx-dep")]
-    /// Convert [ConnectOptions] into [sqlx::pool::PoolOptions]
-    pub fn pool_options<DB>(self) -> sqlx::pool::PoolOptions<DB>
```

Comment:
This seems like a breaking changes. Can you list out all other breaking changes?

Reply:
Please check :) https://github.com/SeaQL/sea-orm/pull/2145#issue-2171431425